src/app/core/components/MobileMenu/MobileNavigation.tsx

```diff
@@ -32,6 +35,55 @@ export function MobileNavigation({ handleNavToggle }: IProps) {
       >
         Docs
       </MobileNavigationLink>
+      {user.user.features.lpVaults && (
```

Comment:
it looks like this section should be moved up so its under "Governance" instead of "Docs"